test/test_torch.py

```diff
@@ -13415,6 +13415,26 @@ def test_cat_big(self, device):
         result = torch.cat(concat_list)
         self.assertEqual(result.size(0), SIZE1 + SIZE2)
 
+    def test_index_add(self, device):
```

Comment:
We have a test_index_add already: https://github.com/pytorch/pytorch/blob/master/test/test_torch.py#L2603

Could we put these two tests closer together or maybe consolidate them? Does running that test on CUDA catch this bug (it seems cpu-only)?